pyccl/halos/concentration/ishiyama21.py

```diff
@@ -27,108 +28,35 @@ class ConcentrationIshiyama21(Concentration):
-                    self.c_alpha = 0.32
-        else:  # use profile fitting method
-            if self.relaxed:  # fit only relaxed halos
-                if self.mdef.Delta == 'vir':
-                    self.kappa = 1.22
-                    self.a0 = 2.52
-                    self.a1 = 1.87
-                    self.b0 = 2.13
-                    self.b1 = 4.19
-                    self.c_alpha = -0.017
-                else:  # now it's either 200c or 500c
-                    if int(self.mdef.Delta) == 200:
-                        self.kappa = 0.60
-                        self.a0 = 2.14
-                        self.a1 = 2.63
-                        self.b0 = 1.69
-                        self.b1 = 6.36
-                        self.c_alpha = 0.37
-                    else:  # now it's 500c
-                        self.kappa = 0.38
-                        self.a0 = 1.44
-                        self.a1 = 3.41
-                        self.b0 = 2.86
-                        self.b1 = 2.99
-                        self.c_alpha = 0.42
-            else:  # fit all halos
-                if self.mdef.Delta == 'vir':
-                    self.kappa = 1.64
-                    self.a0 = 2.67
-                    self.a1 = 1.23
-                    self.b0 = 3.92
-                    self.b1 = 1.30
-                    self.c_alpha = -0.19
-                else:  # now it's either 200c or 500c
-                    if int(self.mdef.Delta) == 200:
-                        self.kappa = 1.19
-                        self.a0 = 2.54
-                        self.a1 = 1.33
-                        self.b0 = 4.04
-                        self.b1 = 1.21
-                        self.c_alpha = 0.22
-                    else:  # now it's 500c
-                        self.kappa = 1.83
-                        self.a0 = 1.95
-                        self.a1 = 1.17
-                        self.b0 = 3.57
-                        self.b1 = 0.91
-                        self.c_alpha = 0.26
+        # key: (Vmax, relaxed, Delta)
+        vals = {(True, True, 200): (1.79, 2.15, 2.06, 0.88, 9.24, 0.51),
+                (True, False, 200): (1.10, 2.30, 1.64, 1.72, 3.60, 0.32),
+                (False, True, 200): (0.60, 2.14, 2.63, 1.69, 6.36, 0.37),
+                (False, False, 200): (1.19, 2.54, 1.33, 4.04, 1.21, 0.22),
+                (True, True, "vir"): (2.40, 2.27, 1.80, 0.56, 13.24, 0.079),
+                (True, False, "vir"): (0.76, 2.34, 1.82, 1.83, 3.52, -0.18),
+                (False, True, "vir"): (1.22, 2.52, 1.87, 2.13, 4.19, -0.017),
+                (False, False, "vir"): (1.64, 2.67, 1.23, 3.92, 1.30, -0.19),
+                (False, True, 500): (0.38, 1.44, 3.41, 2.86, 2.99, 0.42),
+                (False, False, 500): (1.83, 1.95, 1.17, 3.57, 0.91, 0.26)}
+
+        key = (self.Vmax, self.relaxed, self.mass_def.Delta)
+        self.kappa, self.a0, self.a1, self.b0, self.b1, self.c_alpha = vals[key]  # noqa
```

Comment:
Stop it with the noqas, break this up.

Reply:
Flake is too strict with that; black, which is stricter, allows 88 chars. :(
I wrapped it.